test/JDBC/expected/Test-sp_execute_postgresql.out

```diff
@@ -0,0 +1,173 @@
+-- tsql
+exec sp_execute_postgresql 'create extension if not exists pg_stat_statements';
+go
```

Comment:
Add negatives tests - no args supplied, arg supplied as NULL, string with 0 length is supplied, some other statements supplied.

Verify search_path, suser_name, dialect is not changes after executing the procedure (with error and w/o error).